clang/lib/Driver/Driver.cpp

```diff
@@ -591,17 +591,6 @@ DerivedArgList *Driver::TranslateInputArgs(const InputArgList &Args) const {
   if (Args.hasFlag(options::OPT_miamcu, options::OPT_mno_iamcu, false))
     DAL->AddFlagArg(nullptr, Opts.getOption(options::OPT_static));
 
-  // Use of -fintelfpga implies -g and -fsycl
```

Comment:
There are few more mentions of 'fpga' in the driver code. Can we please clean them as well?

Thanks

Reply:
The remaining `fpga` usages are for retaining diagnostics for informing users of how these options are not supported and to reference the documentation for more details.  We will keep these checks around until the next major release.

There are also some extension uses in `sycl-post-link` that I kept around, as it is not clear if they are completely safe to remove.  We can remove those when the extensions are cleared from the tool.